pages/my-dashboard.js

```diff
@@ -162,14 +118,10 @@ export default function MyDashboard(props) {
           refPageAA={props.aaPrefix}
         />
       </Modal>
-      <Modal
-        className="flex justify-center bg-black/75 h-full"
-        isOpen={demoModalBody === null ? false : true}
-        onRequestClose={closeModal}
-        contentLabel={'Demo Modal'}
-      >
-        {demoModalBody}
-      </Modal>
+      <SessionTimeTracker
```

Comment:
Just a nitpick so you can ignore this if you want, but would a better name for this component be `SessionTimeout`?